src/game-logic/cards-collection.ts

```diff
@@ -269,6 +270,27 @@ export const cardModels: any = {
     "Invoke by sacrifying a card from your field. Then withdraw a card from your opponent field.",
     simpleAttackInfo
   ),
+  Raid: new CardModel(
+    "Raid",
+    raid,
+    0,
+    0,
+    Color.Red,
+    (card: Card) => {
+      if (
+        card.duel.cards[card.playerId][Zone.Deck].length >= 3 &&
+        card.duel.cards[1 - card.playerId][Zone.Field].length >= 3
+      ) {
+        card.duel.damagePlayer(card.playerId, 3);
+        card.duel.startSelection(1 - card.playerId, Zone.Field);
+        card.duel.destroy(() => card.duel.selectedTarget);
+        card.duel.discard(() => card);
+      }
+    },
+    () => null,
+    "Discard this card. Then draw cards until you draw a card of a color that is in your hand already.",
+    null
```

Comment:
Correct description:
Usable only if your opponent has 3 or more cards on the field. Discard 3 cards from your deck, then select and destroy one card from your opponent's field.